### PR TITLE
Add e2e-azure-ipi-sdn-migration-ovn-p2

### PR DESCRIPTION
**Pull request by weliang1**
@zhaozhanqi @liangxia @jianlinliu Does any one know why `Status=404 Code="ParentResourceNotFound" Message="Can not perform requested operation on nested resource. Parent resource 'ci.azure.devcluster.openshift.com' not found."` keep failing in https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/33542/rehearse-33542-periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-e2e-azure-ipi-sdn-migration-ovn-p2/1585722059441311744?

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 1]
```diff
@@ -1369,6 +1369,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-sdn
+- as: e2e-azure-ipi-sdn-migration-ovn-p2
+  cron: 0 10 */28 * *
+  steps:
+    cluster_profile: azure-qe
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:1375]: Missed `BASE_DOMAIN` overwrite setting.
```
    env:
      BASE_DOMAIN: qe.azure.devcluster.openshift.com
```

**Comment by weliang1** [comment]
@zhaozhanqi @liangxia @jianlinliu `ci/rehearse/periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-e2e-azure-ipi-sdn-migration-ovn-p2` failed is because of the same reason `[[31mcucumber -p junit -p dir_embedder features/upgrade/sdn/multus-upgrade.feature:62[0m[90m # Scenario: Check the multus works well after upgrade[0m](https://issues.redhat.com/browse/OCPBUGS-2665)`, no any workflow error.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 2]
```diff
@@ -1368,6 +1368,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-sdn
+- as: e2e-azure-ipi-sdn-migration-ovn-p2
+  cron: 0 10 */28 * *
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:1372]: p2 runs every 7 days (*/7), and p3 runs every 28 days (*/28). Perhaps make them match each other ?

Comment (weliang1) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml]: @liangxia Fixed it.

**Pull request merged**